server/src/bge/System.cpp

```diff
@@ -199,9 +192,10 @@ namespace bge {
             forwardDirection.y = 0;
             forwardDirection.z = sin(glm::radians(req.yaw));
             forwardDirection = glm::normalize(forwardDirection);
-            req.forwardDirection = forwardDirection;
+            req.forwardDirection = glm::normalize(forwardDirection);
```

Comment:
Isn't this already normalized on the previous line?

Reply:
oh shoot, i must've been blind, or was git merge behaving again?

good catch, fixed